src/sql/workbench/services/connection/browser/connectionDialogWidget.ts

```diff
@@ -198,6 +198,7 @@ export class ConnectionDialogWidget extends Modal {
 		DOM.hide(this._savedConnection);
 
 		this._panel = new TabbedPanel(this._body);
+		this._panel.element.style.margin = '0px 10px';
```

Comment:
Why are you doing this here? Generally I find it useful to keep as many of the styles in the CSS as we can

Reply:
I agree if you are talking about the using CSS to style the current component, but I think we should avoid customizing another component's style in CSS class, in this case, if the class name is changed in the panel component, we will very likely miss this one.